flask/boxwise_flask/auth_helper.py

```diff
@@ -169,9 +171,14 @@ def authorization_test(test_for, **kwargs):
                 401,
             )
 
-
-def test_base(requesting_user, base_id):
-    users_bases = requesting_user["base_ids"]
-    if base_id in users_bases:
-        return True
-    return False
+def user_can_access_base(requesting_user, base_id):
+    if "base_ids" in requesting_user:
+        users_bases = requesting_user["base_ids"]
+        if base_id in users_bases:
+            return SUCCESS
+    else:
```

Comment:
Shouldn't this branch belong to the innermost `if`? If `base_ids` is not present in the user parameters, it would hint to a bad request (400) rather?

Reply:
if this function returns a failure an auth error is raised in the calling function